test/infrastructure/docker/internal/docker/util.go

```diff
@@ -27,10 +27,22 @@ import (
 	"sigs.k8s.io/cluster-api/test/infrastructure/docker/internal/docker/types"
 )
 
-const clusterLabelKey = "io.x-k8s.kind.cluster"
-const nodeRoleLabelKey = "io.x-k8s.kind.role"
-const filterLabel = "label"
-const filterName = "name"
+const (
+	clusterLabelKey  = "io.x-k8s.kind.cluster"
+	nodeRoleLabelKey = "io.x-k8s.kind.role"
+	filterLabel      = "label"
+	filterName       = "name"
+
+	failureDomainLabelKey = "io.x-k8s.kind.failureDomain"
```

Comment:
Are the other two labels above just copies of kind constants? If yes should we use the cluster api namespace for the new label?

Reply:
Yes, they are copies of kind labels.
I don't think there are problems with renaming them, but for this PR I just renamed the new one

Reply:
Oh yup. That was my intention. I'm not sure, but I suspect we shouldn't rename the existing "actual" kind labels because they are used e.g. for `kind delete clusters --all` and I think it's very convenient that that just deletes everything including CAPD clusters :)